react-ui/src/components/Layouts/Dashboard.js

```diff
@@ -62,7 +75,7 @@ class Dashboard extends Component {
           <div className="main">
             <Switch>
               <Route path={`${match.path}/page2`}>
-                <Page2 name={this.state.personId}/>
```

Comment:
Hi Melissa, 
Is personId a unique identified for both students and service staff? I would recommend keeping the identifiers consistent with the backend user login/account creation terminology. Check out the way students and service staff are stored in > MSCI342-Term-Project/server/models/data/Student.js / and MSCI342-Term-Project/server/models/data/Worker.js /